custom/icds_reports/reports/awc_reports.py

```diff
@@ -1012,7 +1012,7 @@ def get_awc_report_beneficiary(start, length, draw, order, filters, month, two_b
     filters['open_in_month'] = 1
     filters['valid_in_month'] = 1
     if filters.get('age_in_months__range') is None:
-        filters['age_in_months__lte'] = 72
```

Comment:
I think this does not need to change. This child beneficiary list intentionally show list of all children  0-6years.

Reply:
ping!!